ci-operator/step-registry/openshift-qe/rosa-loaded-upgrade/openshift-qe-rosa-loaded-upgrade-commands.sh

```diff
@@ -0,0 +1,174 @@
+#!/bin/bash
+set -o errexit
+set -o nounset
+set -o pipefail
+# Log in
+function rosa_login()
+{
+  if [ $# -eq 1 ];then
+    REGION=$1
+  fi
+
+  ROSA_LOGIN_ENV=${ROSA_LOGIN_ENV:="staging"}
+  ROSA_VERSION=$(rosa version)
+  ROSA_TOKEN=$(cat "${CLUSTER_PROFILE_DIR}/ocm-token")
+
+  if [[ ! -z "${ROSA_TOKEN}" ]]; then
+     echo "Logging into ${ROSA_LOGIN_ENV} with offline token using rosa cli ${ROSA_VERSION}"
+     rosa login --env "${ROSA_LOGIN_ENV}" --token "${ROSA_TOKEN}"
+     if [ $? -ne 0 ]; then
+       echo "Login failed"
+       exit 1
+     fi
+  else
+     echo "Cannot login! You need to specify the offline token ROSA_TOKEN!"
+     exit 1
+  fi
+  echo
+  echo "######################################################################"
+  rosa list clusters --region $REGION
+  echo "######################################################################"
+  echo 
+}
+
+function classic_rosa_upgrade()
+{
+
+  SCHEDULE_OFFSET=15  #After $SCHEDULE_OFFSET minutes to upgrade
+  SCHEDULE_DATETIME=`date  -d "+${SCHEDULE_OFFSET} min" "+%Y-%m-%d %H:%M"`
```

Comment:
Do we need to set schedule for specific CHANNEL_GROUP testing? Or is this more for staging vs production upgrade test?

Reply:
ROSA upgrade have to set schedule time, It can not directly trigger the upgrade. ROSA only support upgrade in the same CHANNEL_GROUP, can not across CHANNEL_GROUP